package.json

```diff
@@ -27,7 +27,12 @@
   "stylelint": {
     "extends": "@cerner/stylelint-config-terra",
     "rules": {
-      "scss/at-mixin-pattern": "^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$"
+      "scss/at-mixin-pattern": "^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$",
```

Comment:
can we update these rule changes to only apply to the carbon-graphs package?

Reply:
I think these rules are for the terra-graphs package as carbon-graphs doesn't have scss files